tests/python/pants_test/backend/python/tasks/test_build_local_python_distributions.py

```diff
@@ -115,6 +117,26 @@ def _retrieve_single_product_at_target_base(self, product_mapping, target):
     single_product = all_products[0]
     return single_product
 
+  # TODO: This snapshot version tagging is only performed for python_dist() targets, but the
+  # setup.py invocation command line is generated in SetupPyRunner in setup_py.py. Could this
```

Comment:
And I think we agree on the problem I mentioned above!

Reply:
Indeed! Made the change in 2aa1a5f -- also moved some comments here into docstrings.